UnityGLTF/Assets/Main.unity

```diff
@@ -113,196 +113,73 @@ NavMeshSettings:
       propertyPath: m_LocalRotation.w
       value: 1
       objectReference: {fileID: 0}
-    - target: {fileID: 4000011406631220, guid: 5255039e40b37b14fbcc0b72a30fc62e, type: 2}
+    - target: {fileID: 4225727463385260, guid: 426ae0d14f1f31f48af072147ac6abe9, type: 2}
       propertyPath: m_RootOrder
-      value: 3
-      objectReference: {fileID: 0}
-    - target: {fileID: 114000010638385648, guid: 5255039e40b37b14fbcc0b72a30fc62e,
-        type: 2}
-      propertyPath: Url
-      value: http://localhost:8080/glTF-Sample-Models/2.0/AnimatedCube/glTF/AnimatedCube.gltf
-      objectReference: {fileID: 0}
-    - target: {fileID: 114000010638385648, guid: 5255039e40b37b14fbcc0b72a30fc62e,
-        type: 2}
-      propertyPath: GLTFStandard
-      value: 
-      objectReference: {fileID: 4800000, guid: a5732b1b5b2e36d47a3da75e187eba06, type: 3}
-    - target: {fileID: 114000010638385648, guid: 5255039e40b37b14fbcc0b72a30fc62e,
-        type: 2}
-      propertyPath: Multithreaded
       value: 1
       objectReference: {fileID: 0}
-    - target: {fileID: 114000010638385648, guid: 5255039e40b37b14fbcc0b72a30fc62e,
+    - target: {fileID: 114843937143698750, guid: 426ae0d14f1f31f48af072147ac6abe9,
         type: 2}
       propertyPath: GLTFUri
-      value: 
+      value: http://localhost:8080/glTF-Asset-Generator/Output/Positive/Material/Material_07.gltf
       objectReference: {fileID: 0}
-    - target: {fileID: 4000011406631220, guid: 5255039e40b37b14fbcc0b72a30fc62e, type: 2}
-      propertyPath: m_LocalScale.x
-      value: 1
+    - target: {fileID: 114843937143698750, guid: 426ae0d14f1f31f48af072147ac6abe9,
+        type: 2}
+      propertyPath: UseStream
+      value: 0
       objectReference: {fileID: 0}
-    - target: {fileID: 4000011406631220, guid: 5255039e40b37b14fbcc0b72a30fc62e, type: 2}
-      propertyPath: m_LocalScale.y
+    - target: {fileID: 114843937143698750, guid: 426ae0d14f1f31f48af072147ac6abe9,
+        type: 2}
+      propertyPath: loadOnStart
       value: 1
       objectReference: {fileID: 0}
-    - target: {fileID: 4000011406631220, guid: 5255039e40b37b14fbcc0b72a30fc62e, type: 2}
-      propertyPath: m_LocalScale.z
-      value: 1
+    - target: {fileID: 114819471581835328, guid: 426ae0d14f1f31f48af072147ac6abe9,
+        type: 2}
+      propertyPath: distance
+      value: 1.283
       objectReference: {fileID: 0}
-    - target: {fileID: 114000010638385648, guid: 5255039e40b37b14fbcc0b72a30fc62e,
+    - target: {fileID: 114819471581835328, guid: 426ae0d14f1f31f48af072147ac6abe9,
         type: 2}
-      propertyPath: loadOnStart
-      value: 0
+      propertyPath: m_Enabled
+      value: 1
       objectReference: {fileID: 0}
     m_RemovedComponents: []
-  m_ParentPrefab: {fileID: 100100000, guid: 5255039e40b37b14fbcc0b72a30fc62e, type: 2}
+  m_ParentPrefab: {fileID: 100100000, guid: 426ae0d14f1f31f48af072147ac6abe9, type: 2}
   m_IsPrefabParent: 0
---- !u!114 &10002067 stripped
-MonoBehaviour:
```

Comment:
What are all of these changes in practice?

Reply:
Instead of a custom light, camera, and GLTF component, using the TestScenePrefab that already contains all 3.  The Main scene essentially becomes that prefab used by the test, plus a floor and the parts to easily pick which model you want loaded.

---
In reply to: [358474507](https://github.com/KhronosGroup/UnityGLTF/pull/532#discussion_r358474507) [](ancestors = 358474507)